source/whats-new.txt

```diff
@@ -34,6 +35,26 @@ Learn what's new in:
 * :ref:`Version 6.1 <version-6.1>`
 * :ref:`Version 6.0 <version-6.0>`
 
+.. _version-6.14:
+
+What's New in 6.14
+------------------
+
+The {+driver-short+} v6.14 release includes the following features:
+
+- Adds support for the ``$lookup`` aggregation stage on encrypted collections. You must
+  upgrade your ``mongodb-client-encryption`` package to v6.3 or later to use this feature.
+
+- Removes the need to import the ``util/types`` package by specifying the {+driver-short+}'s
+  internal implementation of the ``isUint8Array`` method.
```

Comment:
S: resolves the vale issue
```suggestion
- Uses the driver's implementation of the ``isUint8Array()`` method to prevent
  importing the ``util/types`` package.
```